packages/web-api/src/WebClient.ts

```diff
@@ -104,7 +104,14 @@ export class WebClient extends EventEmitter<WebClientEvent> {
     this.rejectRateLimitedCalls = rejectRateLimitedCalls;
 
     // Logging
-    this.logger = getLogger(WebClient.loggerName, logLevel, logger);
+    if (typeof logger !== 'undefined') {
+      this.logger = logger;
+      if (typeof logLevel !== 'undefined') {
+        this.logger.debug('The logLevel given to WebClient was ignored as you also gave logger');
```

Comment:
So the `loglevel` parameter is useless in this instance because by passing in logger, it is assumed it already has a `loglevel` set